apps/sms/js/link_helper.js

```diff
@@ -58,7 +58,7 @@ function checkDomain(domain) {
 }
 
 // defines things that can match right before to be a "safe" link
-var safeStart = /[\s,:;\(>]/;
+var safeStart = /[\s,:;\(>\u0080-\uFFFF]/;
```

Comment:
Heh, nice :) Thanks for the explanation, I'm curious if iOS and Android support such numbers